demo/tests.py

```diff
@@ -74,4 +74,19 @@ def test_menu_links(client: TestClient, url: str):
     assert isinstance(data, list)
 
 
+# def test_forms_validate_correct_select_multiple(client: TestClient):
```

Comment:
Is this meant to be commented out?

Reply:
not sure, this is coming from the upstream (https://github.com/pydantic/FastUI/tree/main)